pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -81,6 +82,33 @@ func writeVifFile(buf []byte, pid, name string) error {
 	return nil
 }
 
+func setPodInterfaceCache(iface *v1.Interface, podInterfaceName string, uid string) error {
```

Comment:
this method of ip detection is specific to the pod network interface.  masquerade,bridge,slirp are bindings of network interface (I know you know that, not trying to be condescending, just clarifying for others)

The reason this seemingly only pertains to masquerade at the moment is because a pod interface with bridge binding isn't supported for migrations at the moment. It's possible that will occur at some point. I'm not sure. The point is the ip is associated with the interface and not the binding mechanism.

Reply:
> this method of ip detection is specific to the pod network interface. masquerade,bridge,slirp are bindings of network interface (I know you know that, not trying to be condescending, just clarifying for others)

No problem at all; it is relevant and useful, for the reasons you've stated.

> 
> The reason this seemingly only pertains to masquerade at the moment is because a pod interface with bridge binding isn't supported for migrations at the moment. It's possible that will occur at some point. I'm not sure. The point is the ip is associated with the interface and not the binding mechanism.

Fair enough; I would then politely ask for the PR's text / commit msgs to be updated reflecting that. 

It's easy to assume we're dealing with a masquerade only problem when the title of the PR is "update latest Pod IP for Masquerade", the PR's description "This adds a way to update VMI Status for Masquerade bindings with latest Pod IP" and the only text in the commit is "Add a way to update VMI Status for Masquerade binding with latest Pod IP".

I feel mislead :)

Reply:
@maiqueb Sorry for the confusion, I updated the description and title, seems better?